employeeService/util/db/employee_database_util.bal

```diff
@@ -40,11 +40,16 @@ public function insertData (string name, string age, string ssn, string employee
     }
     // Initialize update status as unsuccessful MySQL operation
     json updateStatus = {"Status":"Data Not Inserted"};
-
-    string sqlString = "INSERT INTO EMPLOYEES (Name, Age, SSN, EmployeeID) VALUES ('" + name + "','" + age + "','" +
-                       ssn + "','" + employeeId + "')";
+    // Prepare the sql string with employee data as parameters
+    sql:Parameter[] params = [];
+    sql:Parameter para1 = {sqlType:sql:Type.VARCHAR, value:name};
+    sql:Parameter para2 = {sqlType:sql:Type.VARCHAR, value:age};
+    sql:Parameter para3 = {sqlType:sql:Type.VARCHAR, value:ssn};
+    sql:Parameter para4 = {sqlType:sql:Type.VARCHAR, value:employeeId};
```

Comment:
Why all the fields are VARCHAR here? According to [1] it seems only the name is varchar and others should be INTEGER

Reply:
Yes, I noticed that and have changed it now